### PR TITLE
Monthly default experiment: Remove experiment code

### PR DESCRIPTION
**Pull request by niranjan-uma-shankar**
#### Changes proposed in this Pull Request

* Removes experiment code introduced in https://github.com/Automattic/wp-calypso/pull/60182 and https://github.com/Automattic/wp-calypso/pull/59771. Control experience is retained.

#### Testing instructions

* Go through the signup flow and verify that you are able to complete purchase of a plan. No JS errors should be seen in the console.
* On an account which has the legacy plans, verify that the logged-in `/plans` page in Calypso loads without JS errors, and that you are able to upgrade to a higher (legacy) plan.

### CODE REVIEW

File: client/my-sites/checkout/upsell-nudge/index.jsx [hunk 1]
```diff
@@ -63,7 +56,6 @@ import {
 	hasLoadedStoredCardsFromServer,
 } from 'calypso/state/stored-cards/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
-import { AnnualPlanUpgradeUpsell } from './annual-plan-upgrade-upsell';
 import { BusinessPlanUpgradeUpsell } from './business-plan-upgrade-upsell';
 import PurchaseModal from './purchase-modal';
 import { extractStoredCardMetaValue } from './purchase-modal/util';
```
Comment (claudiucelfilip) [inline client/my-sites/checkout/upsell-nudge/index.jsx:61]: `export const ANNUAL_PLAN_UPGRADE` looks likes is have [been defined](https://github.com/Automattic/wp-calypso/pull/59771/files#diff-3283d25431f847b8456e3f5532f3e88cca8024d24dfc365def5eec8b9d7d80ddR81) for the experiment on line 76. 
Should we remove it?

**Comment by sirbrillig** [comment]
Does this have anything to do with these properties added by https://github.com/Automattic/wp-calypso/pull/46466? I've been wondering about them recently since they say they are temporary:

https://github.com/Automattic/wp-calypso/blob/6b10cb05861c010802a76b089acc32deea84848c/packages/shopping-cart/src/types.ts#L336-L338

cc @taggon

**Comment by niranjan-uma-shankar** [comment]
> Does this have anything to do with these properties added by https://github.com/Automattic/wp-calypso/pull/46466?

This PR is related to making monthly plans tab the default for India traffic, introduced by #59771 and is not related to #46466.

File: client/my-sites/checkout/upsell-nudge/index.jsx (hunk 1 above)
Comment (niranjan-uma-shankar) [reply client/my-sites/checkout/upsell-nudge/index.jsx:61]: Good catch, I've pushed in a change to remove it.

**Review by claudiucelfilip — APPROVED** [review]
LGTM🚀

**Pull request merged**